packages/matchbox-site/src/components/MDXProvider.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import { MDXProvider } from '@mdx-js/react';
 import { Link } from 'gatsby';
 import { PropsTable } from './PropsTable';
+import { ComponentProps } from './ComponentProps';
```

Comment:
Instead of a component for all the props, might need to export a singular `<Prop>` component so we can control content in the mdx pages